### PR TITLE
find by status

### PR DESCRIPTION
**Pull request by GMSzebeni**
find by status function added with tests

### CODE REVIEW

**Comment by BlazsejKrisz** [comment]
Nice

**Review by BlazsejKrisz — APPROVED** [review]

File: src/app.js [hunk 1]
```diff
@@ -15,10 +15,24 @@ export function createApp(todoStore, args) {
...
-      const validated = validateAddParams(params);
-      const added = add(todoStore, validated);
+      const validatedAdd = validateAddParams(params);
+      const added = add(todoStore, validatedAdd);
       display(['New Todo added:', format(added)])
       break;
+    case 'find-by-status':
+      const validatedStatus = validateFindByStatusParam(params);
+      const filteredTodos = findByStatus(todoStore, validatedStatus);
+      if(filteredTodos.length === 0) {
```
Comment (kepelivi) [inline src/app.js]: Please move the array length checking to todo.js.

**Review by kepelivi — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 1 above)
Comment (GMSzebeni) [reply src/app.js]: moved it :)

**Review by kepelivi — APPROVED** [review]
Thank you for completing the change, everything is ok!

**Pull request merged**